release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,16 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 29 February 2024 - New Feature - Admin - Organization keys added [ID_38944]
+
+The [Admin app](https://admin.dataminer.services) now includes organization keys. A user can create an organization-specific key with certain permissions.
```

Comment:
Do you mean that if you have certain permissions, you can create an organization-specific key, or that you can create an organization-specific key that comes with certain permissions?

Reply:
Removed this as it's been merged by another PR